faq.md

```diff
@@ -155,7 +162,7 @@ fn is_odd(x: i64) -> bool {
 }
 ```
 
-In each example, the last line of the function is the return value of that function. It is important to note that if a function ends in a semicolon, its return type will be `()`, indicating no returned value. Implicit returns must omit the semicolon to work.
+In each example, the last line of the function is the return value of that function. It is important to note that if a block ends in a semicolon, its return type will be `()`, indicating no returned value. Implicit returns must omit the semicolon to work.
```

Comment:
Hm, good point. To be honest I've never found the concept of "implicit return" to be a helpful way to think about what's going on here. Anyway, I'll back out this particular change.